torch_xla/csrc/convolution.cpp

```diff
@@ -234,6 +234,16 @@ xla::XlaOp BuildThnnConv2dBackwardWeight(
                         {3, 2, 0, 1});
 }
 
+xla::XlaOp BuildGradBias(xla::XlaOp grad_output) {
+  xla::Shape grad_output_shape = XlaHelpers::ShapeOfXlaOp(grad_output);
+  return xla::Reduce(
+      grad_output,
+      XlaHelpers::ScalarValue<float>(0, grad_output_shape.element_type(),
+                                     grad_output.builder()),
+      XlaHelpers::CreateAddComputation(grad_output_shape.element_type()),
+      {0, 2, 3});
```

Comment:
Would be good to have some comment around these magic reduce dimensions. Same for the Transpose 10 LOC above.

Reply:
Done.

Reply:
Done.